### ISSUE TITLE
Create contrarian agent

### ISSUE THREAD

**Issue by gabrielfior** [gnosis/prediction-market-agent#290]
It's been documented elsewhere (https://github.com/gnosis/prediction-market-agent/issues/280) that the betting profitability of an agent depends on the market odds at the time of the bet. This is not hard to follow - if team A plays team B and team A currently has higher odds, betting on the other team yields a larger payoff than betting on team A.

The idea of this task is to create an agent that solely bets on unbalanced markets (e.g. 70% odds for yes) and the agent thinks NO is the correct answer.
I would expect this to bet on a small quantity of markets but with high conviction (hopefully).

**Comment by alramalho** [comment]
sounds great hehe. as a user i would be mostly curious about trying this one specifically. also, a general score that weights agents risk in a general waywould be interesting.

i.e. if we just evaluate how does an agent perform in terms of whether their bet held true, a low risk agent will always outpreform a high risk agent, even though from the traditional 'betting' standpoint the high risk agent could be better

**Comment by gabrielfior** [comment]
@alramalho thanks for the comment.
Can you further expand into the score you mentioned? In my mind, I think what you are alluding to is similar to the payoff of the agent, i.e. how profitable is it - but not sure if this corresponds to your question.
